src/components/immersive/byline.tsx

```diff
@@ -27,22 +27,22 @@ const Styles = ({ kicker }: PillarStyles): SerializedStyles => css`
 
         time {
             font-size: 1.4rem;
-            color: ${palette.neutral[46]};
+            color: ${neutral[46]};
         }
     }
 `;
 
 const DarkStyles = ({ inverted }: PillarStyles): SerializedStyles => darkModeCss`
-    background: ${palette.neutral.darkMode};
-    color: ${palette.neutral[86]};
+    background: ${background.inverse};
+    color: ${neutral[86]};
 
     .author {
         .follow, a {
             color: ${inverted};
         }
 
         time {
-            color: ${palette.neutral[60]};
+            color: ${neutral[60]};
```

Comment:
I'm going to investigate adding the following tokens:

- neutral[86] -> text.inversePrimary`
- neutral[60] -> text.inverseSecondary`

Would this make sense, from your perspective?

Reply:
Ah, yes, I've actually made a note to talk to you about those two colours. If you're planning to create a semantic "dark mode" name for them that would be great!